application/features/SFTP.py

```
@@ -59,6 +63,7 @@
                 self.sftp.chdir(path)
             cwd = self.sftp.getcwd()
             attrs = self.sftp.listdir_attr(cwd)
+            logger.debug(f"SFTP: ls {cwd}: {attrs}")
```

Ruff

Logging statement uses f-string

(G004)